azurerm/resource_arm_monitor_action_group_test.go

```diff
@@ -440,38 +930,328 @@ resource "azurerm_monitor_action_group" "test" {
   resource_group_name = "${azurerm_resource_group.test.name}"
   short_name          = "acctestag"
 
-  email_receiver {
-    name          = "sendtoadmin"
-    email_address = "admin@contoso.com"
-  }
+  automation_runbook_receiver {
+		name = "action_name_1"
+		automation_account_id = "${azurerm_automation_account.test.id}"
+  	runbook_name = "${azurerm_automation_runbook.test.name}"
+		webhook_resource_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-runbooks/providers/microsoft.automation/automationaccounts/aaa001/webhooks/webhook_alert"
+		is_global_runbook = true
+		service_uri = "https://s13events.azure-automation.net/webhooks?token=randomtoken"
+	}
+}
 
-  email_receiver {
-    name          = "sendtodevops"
-    email_address = "devops@contoso.com"
-  }
+resource "azurerm_automation_account" "test" {
+	name                = "aaa001"
```

Comment:
This is going to cause issues with running tests in parallel, could we change it to be random?
```suggestion
	name                = "acctestAA-%d"
```

Reply:
Thanks, @katbyte!  I followed that naming scheme for all entries.  The only one with Azure length limits are the storage accounts for the function app at 24 characters, which I shortened, hopefully correctly.  In the go playground example, I could divide an 18-digit int by 1e12 and fmt specify it as %6.0f, but 'go vet' complained about that and wanted rInt/1e12 as an int.